spec/css/media/logic/error.hrx

```diff
@@ -0,0 +1,270 @@
+<===> and_after/type_and_not/input.scss
+@media a and not (b) and (c) {x {y: z}}
+
+<===> and_after/type_and_not/error
+Error: expected "{".
+  ,
+1 | @media a and not (b) and (c) {x {y: z}}
+  |                      ^
+  '
+  input.scss 1:22  root stylesheet
+
+<===>
+================================================================================
+<===> and_after/or/input.scss
+@media (a) or (b) and (c) {x {y: z}}
```

Comment:
Should there be a test (maybe under `nested`?) where you combine and/or/not on different levels?

e.g. `@media (a) or ((b) and (c))`

Reply:
Good thought.